sshd/src/main/scala/ammonite/repl/sshd/ShellSession.scala

```diff
@@ -0,0 +1,63 @@
+package ammonite.repl.sshd
+
```

Comment:
Every file should have the [Acyclic Plugin](https://github.com/lihaoyi/acyclic) enabled via `import acyclic.file`.